### ISSUE TITLE
Multiple refunds not working

### ISSUE THREAD

**Issue by raguventhan** [razorpay/razorpay-magento#279]
In my Magento 2 store, only one time i can initiate refund for single invoice.

Description: If invoice contains 2 or more items and if i refund for one item, again i can't initiate second refund for the remaining items

**Comment by ramth05** [comment]
Yes, this is the current limitation in the plugin. We will come back soon on this.
